README.md

```diff
@@ -23,10 +23,10 @@ Example output:
     class HelloComponent extends Component {
       render() {
         return React.createElement("div", {
-          "data-element": "div",
-          "data-component": "HelloComponent"
+          "data-component": "HelloComponent",
+          "data-file-source": "hello-component.js"
         }, React.createElement("h1", {
-          "data-element": "h1"
+          null
```

Comment:
@TrevorFSmith I think it would be helpful to have an additional example of the rendered HTML as well.